File: 441-arranging-coins/441-arranging-coins.cpp
class Solution {
public:
    int arrangeCoins(int n) 
    {
        int i=1;
        int count=0;
        while(n>0)
        {
            n=n-i;
            if(n>=0) ++count;
            else break;
            ++i;
        }   
        return count;
    }
};

File: 441-arranging-coins/441-arranging-coins.py
class Solution:
    def arrangeCoins(self, n: int) -> int:
        rows,count=1,0
        while n>0:
            n=n-rows
            if n>=0:
                count=count+1
            else:
                break
            rows=rows+1
        return count   
    
        